src/fabric-expandable/index.js

```diff
@@ -0,0 +1,43 @@
+import { getIframeId, resizeIframeHeight, sendMessage } from '../_shared/js/messages';
+import { enableToggles } from '../_shared/js/ui';
+import { write, div } from '../_shared/js/dom';
+
+getIframeId()
+.then(() => {
+    handleScroll();
+    handleToggle();
+});
+
+function handleScroll() {
+    let isMobile = window.matchMedia('(max-width: 739px)').matches;
```

Comment:
That's `740 - 1`, and `740` is the `mobile/tablet` breakpoint width. Admittedly this is not very clear.